api/routes/apds/get.js

```diff
@@ -92,8 +85,7 @@ module.exports = (
             id,
             created,
             updated,
-            state,
-            validation
```

Comment:
why take this stuff out?

Reply:
I only put it in for proof of concept and I already demoed it. I've created a [wiki page](https://github.com/CMSgov/eAPD/wiki/LaunchDarkly) for how to use it.